authors/apps/articles/models.py

```diff
@@ -17,41 +16,23 @@ class Article(models.Model):
 
     created_at = models.DateTimeField(auto_now_add=True)
 
-    updated_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
 
     favorited = models.BooleanField(default=False)
 
     favorites_count = models.IntegerField(default=0)
 
-
     author = models.ForeignKey(
         User,
         related_name='articles',
         on_delete=models.CASCADE)
 
-    read_time = models.TextField()
+    read_time = models.CharField(max_length=255)
```

Comment:
Kindly refrain from adding changes not related to a story you pick up.
If you feel this is a fix that needs addressing, get in touch with the TTL or PO to create a story for it.

Reply:
Ohh, am sorry about this. This change came as a result when I was testing since at first I thought that read_time field was storing a body field during its calculation when determining time. So I decided to restrict its characters and use a body of characters greater than 255 characters to see whether I could get any errors. But later I figured out that its use was just to display the time of reading. Then mistakenly I commited it with my updates but it wasn't intentional to change it. But it's now fixed.